examples/crowd-funding/README.md

```diff
@@ -71,8 +75,8 @@ Alternatively, the command below can be used to list the chains created for the
 known by each wallet:
 
 ```bash
-linera --wallet "$LINERA_WALLET_0" --storage "$LINERA_STORAGE_0" wallet show
-linera --wallet "$LINERA_WALLET_1" --storage "$LINERA_STORAGE_1" wallet show
+linera --with-wallet 0 wallet show
```

Comment:
FWIW `-w 0` or even `-w0` also works.

Reply:
I thought about it but for pædagogic reasons I decided the longer form was probably preferable — since the readers of this are unlikely to be very familiar with Linera, it's best not to overburden their working memory.